analysis_options.yaml

```diff
@@ -21,13 +22,18 @@ linter:
     - avoid_returning_this
     - avoid_setters_without_getters
     - avoid_slow_async_io
+    - avoid_unused_constructor_parameters
     - cancel_subscriptions
     - cast_nullable_to_non_nullable
+    - collection_methods_unrelated_type
     - comment_references
     - dangling_library_doc_comments
+    - depend_on_referenced_packages
```

Comment:
This one is already included in core so can be safely removed I think.

Reply:
Nice catch.